jsonp/2.0/_index.md

```diff
@@ -11,13 +11,13 @@ querying JSON documents.
 * [Jakarta JSON Processing 2.0 Specification Document]() (PDF)
```

Comment:
These two empty links should be removed, or link to a boilerplate spec document like for the previous version.